pyproject.toml

```diff
@@ -23,7 +23,6 @@ description = "AI assisted Virtual Laboratory framework."
 
 dependencies = [
     "numpy >= 1.20.0",
-    "tk >= 0.1.0",
```

Comment:
Removed `tk` as dep - we thought it installed Tkinter, but actually installs a different package. 

`TKinter` is installed as part of the base python package, but for python3.10+, you need to install `python3.10-full` rather than just `python3.10` to install it. Added a troubleshoot to the wiki to reference this